packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStart/ForceFunctionContainerStartExecutor.test.ts

```diff
@@ -314,10 +314,10 @@ describe('ForceFunctionContainerStartExecutor unit tests', () => {
 
       assert.calledOnce(fakeDisposible.dispose);
       assert.calledTwice(localizeStub);
-      console.log('what are the args', {
-        args0: sendExceptionStub.getCalls()[0].args[0],
-        args1: sendExceptionStub.getCalls()[0].args[1]
-      });
+      // console.log('what are the args', {
+      //   args0: sendExceptionStub.getCalls()[0].args[0],
+      //   args1: sendExceptionStub.getCalls()[0].args[1]
+      // });
```

Comment:
What do you think about either adding a comment here as to why this is no longer applicable or removing entirely?

Reply:
I'll remove them.  This was there and was adding noise to the output.  salesforcedx-vscode-core has a lot of noise in it when the untegration tests run.  When the tests run (either locally, or in CI) there are lots of errors reported, and it's at the point where it's hard to tell when a failure is a failure or not.  This is just a "drop in the ocean" and I hope to focus on this at some point and clean up the build.